handlers/handlers.go

```diff
@@ -63,15 +63,15 @@ func isProtectedCookie(stringToFind string) bool {
 func removeNonProtectedCookies(w http.ResponseWriter, req *http.Request) {
 	for _, cookie := range req.Cookies() {
 		if !isProtectedCookie(cookie.Name) {
-			cookie := &http.Cookie{
+			cookie2 := &http.Cookie{
```

Comment:
I assume you're renaming this as the linter has complained that variable name is the same as the value? However `cookie2` is not a great time; I suggest either `setCookie` or pass through a slice 
```go
			http.SetCookie(w, &http.Cookie{
				Name:     cookie.Name,
				Value:    "",
				Path:     "/",
				Expires:  time.Unix(0, 0),
				MaxAge:   0,
				HttpOnly: false,
			})
```

Reply:
Thanks, i'll go with 'setCookie'.